frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -423,6 +423,13 @@
                         vm.phases.results[i].showPrivate = true;
                     }
                 }
+                for(var i=0;i<vm.phases.results.length;i++){
+                    var qwerty=new Date(vm.phases.results[i].start_date).toString();
+                    vm.phases.results[i].start_zone=qwerty.split('(')[1].split(')')[0];
```

Comment:
Please refrain from doing such extraction..Please make use of moment js which we are already using in our project.
FYI: Extracting timezone is pretty straight forward with moment: https://momentjs.com/timezone/